test-apps/display-test-app/src/frontend/Viewer.ts

```diff
@@ -406,7 +406,7 @@ export class Viewer extends Window {
 
   private async resetIModel(filename: string): Promise<void> {
     let newIModel: IModelConnection;
-    const sameFile = filename === this._imodel.getRpcProps().key;
+    const sameFile = filename === this._imodel.key;

```

Comment:
Did I misread or is `key` a hashed filename and so this will no longer work?

Reply:
You're right, I removed this because openStandaloneIModel should now work with the "same file" (it returns the existing Db). But, i don't know how to test this.

Reply:
I added this because previously the following would produce an error - something from Raman's briefcase cache IIRC, so possibly fixed by your changes:
1. Open an iModel.
2. Click "open iModel" toolbar button and choose same iModel.
3. Observe cryptic error; iModel fails to open.

Reply:
I made it supply the filename as the key (so it can show it on the titlebar.) So, I put openStandaloneIModel back the way it was.